include/antidote.hrl

```diff
@@ -230,6 +233,6 @@
 %% clocksi_read_item_fsm as pointers to the materializer's ets caches
 -record(mat_state, {
 	  partition :: partition_id(),
-	  ops_cache :: cache_id(),
-	  snapshot_cache :: cache_id(),
-	  is_ready :: boolean()}).
+	  ops_cache :: atom() | cache_id(),
```

Comment:
In `clocksi_readitem_fsm` this field is initialized with a value from `materializer_vnode:get_cache_name`, which constructs an atom dynamically. So if there is a type that says the atom ends with `"snapshot_cache"`, then we could make it more precise. 

I think the `cache_id()` alternative could be removed.